### ISSUE TITLE
Week 3 lab final changes?

### ISSUE THREAD

**Issue by arthur-e** [OscarFHC/EAS538_2018#13]
Hi @mehajain13 @OscarFHC,

I heard last night that there are some additions to this week's lab. Looking forward to seeing those soon.

I also realized that we don't have an explanation of the fields in the `apdata` data frame. I went to the [USDA website](https://www.ams.usda.gov/datasets/pdp/pdpdata) and tried to access [the data dictionary](https://origin-edit.ams.usda.gov/sites/default/files/media/PDPSearchAppDataDictionary.pdf) but it is not loading right now ("The site can't be reached"). So, this is my best guess:

- `commod` -- "Commodity"? At any rate, everything single row has the value "AP" (Agricultural Product?);
- `concen` -- Most likely "Concentration" of pesticide;
- `lod` -- Some kind of outcome/ response variable related to pesticide use but I have no idea what.

It might be good to give students some background on these data.

**Comment by mehajain13** [comment]
Thanks for checking in Arthur. The addition to the lab will be minimal (a few more questions in the last exercise) and I'll add those in when I get to the office this AM. I'll also add some text on the variable names. Thanks!

M

**Comment by mehajain13** [comment]
Hey! I made a few small changes to the lab and 'pushed' it to Oscar's main directory. I think the new version is ready to go. Specific changes are:
- removed read.csv line since I want students to remember how to do this on their own
- added variable info
- reworded the CI section a bit (first few paragraphs) to match the terminology I used in lecture
- added an additional question to exercise 3 (about sample size + CIs)
Thanks! hope all goes smoothly today.
